dpdd.tex

```diff
@@ -193,14 +192,16 @@ \subsection{Classes of LSST Data Products}
 
 
 \begin{enumerate}
-\item Analysis of difference images, with the goal of detecting and characterizing astrophysical phenomena revealed by their time-dependent nature. The detection of supernovae superimposed on bright extended galaxies is an example of this analysis. The processing will be done on nightly or daily basis and result in \textbf{Prompt} data products. Prompt products will include difference images, catalogs of sources detected in difference images (\DIASources), astrophysical objects\footnote{The LSST has adopted the nomenclature by which single-epoch detections of astrophysical \emph{objects} are called \emph{sources}. The reader is cautioned that this nomenclature is not universal: some surveys call \emph{detections} what LSST calls \emph{sources}, and use the term \emph{sources} for what LSST calls \emph{objects}.} these are associated to (\DIAObjects), and Solar System objects (\SSObjects\footnote{\SSObjects used to be called ``Moving Objects'' in previous versions of the LSST Data Products baseline. The name is potentially confusing as high-proper motion stars are moving objects as well. A more accurate distinction is the one between objects \emph{inside} and \emph{outside} of the Solar System.}). The catalogs will be entered into the \textbf{\DB} and made available in near real time. Notifications (``alerts'') about new \DIASources will be issued using community-accepted
+    \item \newtext{\textbf{Prompt Processing}, which is performed on a nightly or daily basis and results in \textbf{Prompt Data Products}. This consists primarily of analysis of difference images, with the goal of detecting and characterizing astrophysical phenomena revealed by their time-dependent nature.} The detection of supernovae superimposed on bright extended galaxies is an example of this analysis.  Prompt products will include difference images, catalogs of sources detected in difference images (\DIASources), astrophysical objects\footnote{The LSST has adopted the nomenclature by which single-epoch detections of astrophysical \emph{objects} are called \emph{sources}. The reader is cautioned that this nomenclature is not universal: some surveys call \emph{detections} what LSST calls \emph{sources}, and use the term \emph{sources} for what LSST calls \emph{objects}.} these are associated to (\DIAObjects), and Solar System objects (\SSObjects\footnote{\SSObjects used to be called ``Moving Objects'' in previous versions of the LSST Data Products baseline. The name is potentially confusing as high-proper motion stars are moving objects as well. A more accurate distinction is the one between objects \emph{inside} and \emph{outside} of the Solar System.}). The catalogs will be entered into the \textbf{\PPDB} and made available in near real time. Notifications (``alerts'') about new \DIASources will be issued using community-accepted
```

Comment:
I think I'd drop the “primarily... of difference images” here. Although most of the prompt products are measured on diffims, it's not obvious that the ISR and single frame processing on direct images aren't “primary” (and, of course, we do return quantities measured on the direct images too).

Could you just make this “Prompt Data Products are generated with the goal of rapidly detecting and characterizing astrophysical phenomena ...”?

Reply:
That reads much better; I like that a lot.